rinex/src/tests/production.rs

```diff
@@ -1,24 +1,21 @@
 #[cfg(test)]
 mod test {
-    use crate::tests::toolkit::compare_with_panic;
+    use crate::tests::toolkit::{compare_with_panic, random_name};
     use crate::*;
     fn testbench(path: &str) {
         // parse this file
         let rnx = Rinex::from_file(path).unwrap(); // already tested elsewhere
-        let copy_path = path.to_owned() + "-copy";
-        assert_eq!(rnx.to_file(&copy_path).is_ok(), true); // test writer
-        let copy = Rinex::from_file(&copy_path);
+        let tmp_path = format!("test-{}.rnx", random_name(5));
```

Comment:
In `gdal` we use `tempfile` and something like https://github.com/georust/gdal/blob/c7f121c01748e92a271d960e7e48415c186a6c13/src/test_utils.rs#L11 for this.